website/docs/components/form/primitives/partials/code/how-to-use.md

```diff
@@ -69,6 +69,47 @@ Interactive elements in text (associated with the input through `aria-describedb
 </Hds::Form::HelperText>
 ```
 
+### Form::CharacterCount
+
+The default invocation requires a `controlId` argument referencing a valid `<input>` or `<textarea>` element.
+
+The `controlId` value is used to generate an ID, prefixed with `character-count-`, so that the ID can be referenced in the `aria-describedby` attribute of the form control.
+
+```handlebars
+<input type="text" aria-label="input with default character count" id="input-character-count-default"/>
+<Hds::Form::CharacterCount @controlId="input-character-count-default"/>
+```
+
+If the user input needs to be limited to a certain number of characters use `@maxLength`.
```

Comment:
Should we mention that this technically doesn't prevent the user from entering more characters than the `@maxLenght` value, and that it's left to the consumers to implement this functionality (and suggest how to possibly do it using the `maxLength` attribute)?

Reply:
2d2f9e398